docs/reference/sparql-reference.md

```diff
@@ -0,0 +1,280 @@
+# Reference templates for SPARQL queries 
+
+This document contains template SPARQL queries that can be adapted. 
+Comments are added in-code with `#` above each step to explain them so that queries can be spliced together
+
+## Checks/Report generation
+
+### All terms native to ontology 
+note: we assume that all native terms here have the same namespace - that of the ontology
+
+```SPARQL
+# select unique instances of the variable
+SELECT DISTINCT ?term
+WHERE {
+  # selecting where the variable term is either used as a subject or object 
+  { ?s1 ?p1 ?term . }
+  UNION
+  { ?term ?p2 ?o2 . }
+  # filtering out only terms that have the MONDO namespace (assumed to be native terms)
+  FILTER(isIRI(?term) && (STRSTARTS(str(?term), "http://purl.obolibrary.org/obo/MONDO_")))
+}
+```
+
+### Definition lacks xref 
+adaptable for lacking particular annotation
+
+```SPARQL
+# adding prefixes used
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
```

Comment:
always use `prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>`, not `oio`